### ISSUE TITLE
IOS9 Failure

### ISSUE THREAD

**Issue by Daz66** [devgeeks/Canvas2ImagePlugin#58]
Hi,

I recently 'upgraded' my iphone to ios9 and this plugin no longer seems to work.
Is this a known issue and can anyone suggest a workaround?

Thanks in advance,
Darren

**Comment by lavinachitara** [comment]
Yes I am also facing the same issue in iPad after upgrading ios 8 to 9. Is there any security issue as in iOS9, apple guys integrated lots of security things, so this plugin is not supporting to new version? Is there any issue in plugin which need to make it compatible with iOS9? 

Thanks in advance, your help would be appreciated as it is very urgent and application is already in app store.

**Comment by Createstandard** [comment]
Working on my iPhone still but it's built with older Cordova - what version of Cordova are you using?

**Comment by Daz66** [comment]
Hi,

Sorry for the slow reply on this!
I'm using the most recent versions of all the build tools and platforms.

I think the issue is caused by the recent security changes for ios9 - or perhaps the WKWebView clashing with this plugin.

Anyway, for my needs I worked around the problem by loading the image to download into a browser, then the user can download it from there. Pity tho - I preferred the saving canvas to the gallery solution. :(

Cheers, Darren
